tests/test_acasclient.py

```diff
@@ -1118,6 +1118,17 @@ def test_032_advanced_search_ls_things(self):
                                        max_results=1000)
         self.assertIn(codes[0], return_codes)
 
+        # Use 'codetable' data 'format'; 'codes_only' is False
+        results = self.client\
+            .advanced_search_ls_things('project', 'project', 'active',
+                                       value_listings=value_listings,
+                                       codes_only=False,
+                                       max_results=1000,
+                                       combine_terms_with_and=True,
```

Comment:
This test is fine. But a more representative test would be to replace `'active'` with `None` in this call, and instead move `active` into the value listings like so:
```
value_listings[0]["value"] = 'active'
```

It'll be doing the same search but if you expanded out to multiple terms to query:
project status : active
is restricted: false

Then you would want the `'active'` and `'false'` terms to be tied specifically to their respective values.

Reply:
Thanks! I have updated tests to cover `combine_terms_with_and` too.